ops/scripts/update-op-geth.py

```diff
@@ -22,7 +18,7 @@ def update_mod(project):
 		'mod',
 		'edit',
 		'-replace',
-		f'github.com/ethereum/go-ethereum@{GETH_VERSION}=github.com/ethereum-optimism/op-geth@optimism'
+		f'github.com/ethereum/go-ethereum=github.com/rsksmart/op-geth@11a70c76751354d7e82def14a56f89f08e59d7b7' # represents tip of github.com/rsksmart/op-geth#rsk/poc-v0
```

Comment:
Here we use the commit while in other parts we just use the branch. Let's use the same approach if possible

Reply:
having said that, I can create a tag and we set it to that